### PR TITLE
Remove extra slash and fix code blocks

### PR DESCRIPTION
**Pull request by swethakundaram-wm**
It contains bulk changes. These have been manually verified. 

- Fixing some 404s.
- Adding missing images
- Removing extra slash 
- Removing duplicate files

### CODE REVIEW

**Comment by prasanth-reddy-wavemaker-com** [comment]
Here are the changed files

  https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/services/database-services/download-jdbc-driver-jar
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/variables/database-apis
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/variables/events
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/variables/java-services
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/basic/anchor
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/basic/search
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/basic/tree
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/datalive/datatable/datatable-events-methods
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/datalive/form/button
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/datalive/form/color-picker
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/datalive/form/rating-widget
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/datalive/form/toggle
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/datalive/list/list-properties-events-methods
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/dropdown-menu
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/form-widget/button
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/form-widgets/calendar
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/form-widgets/checkbox
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/form-widgets/checkboxset
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/form-widgets/color-picker
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/form-widgets/currency
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/form-widgets/date-time-datetime
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/form-widgets/number
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/form-widgets/select-locale
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/form-widgets/select
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/form-widgets/slider
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/form-widgets/text
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/form-widgets/textarea
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/form-widgets/toggle
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/mobile-widgets/barcode-scanner
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/navigation/dropdown-menu
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/navigation/nav-basic-usage
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/navigation/nav
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/navigation/popover
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/prefab/oauth-prefabs/instagram
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/prefab/qrcode
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/prefab/youtube
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnapp-development/widgets/use-cases-prefabs
        https://pr-74.dpwfbpxczhwpr.amplifyapp.com/learnbuilding-rating-widget-using-variable

File: learn/app-development/services/database-services/download-jdbc-driver-jar.md [hunk 1]
```diff
@@ -2,12 +2,23 @@
...
+2. Select language and click on **Download**.
+3. Select and download `sqljdbc_4.2.8112.100_enu.tar.gz`.
+4. Once the download is complete, extract the downloaded file. Select and upload `sqljdbc42.jar`   
+    Located : [your extract folder] `\sqljdbc_4.2.8112.100_enu\sqljdbc_4.2\enu\jre8` 
+
+:::note
+It is recommended to use **`sqljdbc42.jar`** or any Java ver 8 compatible jar file. 
+:::
+
+[![](/learn/assets/jdbc_driver_step4.png)](/learn/assets/jdbc_driver_step4.png)
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/services/database-services/download-jdbc-driver-jar.md]: This image url still fails to load

File: learn/app-development/widgets/datalive/form/button.md [hunk 2]
```diff
@@ -15,17 +15,17 @@ The button widget can trigger any of the following actions when it is clicked.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/datalive/form/button.md]: This page does not have sidebar/leftnav

File: learn/app-development/widgets/datalive/form/color-picker.md [hunk 3]
```diff
@@ -9,16 +9,15 @@ id: ""
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/datalive/form/color-picker.md]: This page does not have sidebar/leftnav

File: learn/app-development/widgets/datalive/form/color-picker.md [hunk 4]
```diff
@@ -9,16 +9,15 @@ id: ""
 
 <iframe width="100%" height="200" style="background-color: snow;" allowtransparency="true" src="https://apps.wavemakeronline.com/documentation_snippets/#/ColorPicker">Color Picker</iframe>
 
-# Properties
+## Properties
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/datalive/form/color-picker.md]: Has desktop icon at a weird place. I think this was supposed to indicate that this is only available in web and not in mobile. We should find a better place for this icon.

image

File: learn/app-development/widgets/datalive/form/rating-widget.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ id: ""
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/datalive/form/rating-widget.md]: Missing sidebar/leftnav

File: learn/app-development/widgets/datalive/form/toggle.md [hunk 6]
```diff
@@ -5,17 +5,16 @@ id: ""
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/datalive/form/toggle.md]: Missing sidebar

File: learn/app-development/widgets/dropdown-menu.md [hunk 7]
```diff
@@ -21,17 +21,16 @@ The **Menu Layout** - _horizontal_ or _vertical_ and **Menu Position** can be se
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/dropdown-menu.md]: Missing sidebar

File: learn/app-development/widgets/form-widget/button.md [hunk 8]
```diff
@@ -21,11 +21,11 @@ The button widget can trigger any of the following actions when it is clicked.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/form-widget/button.md]: Missing sidebar

File: learn/app-development/widgets/form-widgets/color-picker.md [hunk 9]
```diff
@@ -10,7 +10,7 @@ id: "color-picker"
 Color Picker is available only for web responsive apps.
 :::
 
-**Color Picker **widget helps in selecting a color and translate to the hex value. The Color Picker Tool can be used to select a color on any image opened on your screen. By clicking a point on an image, you can change the active color to that which is located under the pointer.
+**Color Picker** widget helps in selecting a color and translate to the hex value. The Color Picker Tool can be used to select a color on any image opened on your screen. By clicking a point on an image, you can change the active color to that which is located under the pointer.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/form-widgets/color-picker.md]: Desktop icon at a weird place.
image

File: learn/app-development/widgets/datalive/form/button.md (hunk 2 above)
Comment (prasanth-reddy-wavemaker-com) [reply learn/app-development/widgets/datalive/form/button.md]: This page also does not have leftnav learn/app-development/widgets/use-cases-prefabs

File: learn/app-development/widgets/datalive/form/color-picker.md (hunk 4 above)
Comment (prasanth-reddy-wavemaker-com) [reply learn/app-development/widgets/datalive/form/color-picker.md]: as discussed we should evolve a patter for documenting where a widget can be used (desktop, mobile) and from which WM version it is available.

Comment (swethakundaram-wm) [reply learn/app-development/widgets/datalive/form/color-picker.md]: Removed the icon and the duplicate file as well.

File: learn/app-development/services/database-services/download-jdbc-driver-jar.md (hunk 1 above)
Comment (swethakundaram-wm) [reply learn/app-development/services/database-services/download-jdbc-driver-jar.md]: Added the image from the old site. This should work now.

File: learn/app-development/widgets/datalive/form/button.md (hunk 2 above)
Comment (swethakundaram-wm) [reply learn/app-development/widgets/datalive/form/button.md]: Added the side nav for prefab use cases. 
Deleted the duplicate files for the button documentation.

File: learn/app-development/widgets/datalive/form/rating-widget.md (hunk 5 above)
Comment (swethakundaram-wm) [reply learn/app-development/widgets/datalive/form/rating-widget.md]: This was a duplicate doc. Removed now.

File: learn/app-development/widgets/datalive/form/toggle.md (hunk 6 above)
Comment (swethakundaram-wm) [reply learn/app-development/widgets/datalive/form/toggle.md]: These are all duplicates, so removed them from the repo.

**Pull request merged**